locales/en/moves.json

```diff
@@ -80,5 +80,14 @@
       "confirmation": "Are you sure you want to cancel the move for <strong>{{name}}</strong> to <strong>{{location}}</strong>?",
       "warning": "This will delete this move request."
     }
+  },
+  "confirmation": {
+    "page_title": "Move confirmation for {{name}}",
```

Comment:
Maybe we should use the same term as the heading here: `Move scheduled for {{name}}`.

"Confirmation" is more an internal name we use for this step so better to not surface another term to the user in the page content if we can help it.